src/plugins/crypto.c

```diff
@@ -970,7 +970,7 @@ gboolean bd_crypto_tc_open (const gchar *device, const gchar *name, const guint8
     gint ret = 0;
     guint64 progress_id = 0;
     gchar *msg = NULL;
-    struct crypt_params_tcrypt params = {0};
+    struct crypt_params_tcrypt params = {};
```

Comment:
Could you please add and use something like ``#define ZERO_INIT {}``? That way we can easily change it in the future and also the intention would be clear.

Reply:
Done